### PR TITLE
cherry picking 2nd commit from editing l_progress helper to fix issue#814

### PR DESCRIPTION
**Pull request by pliu2781**

### CODE REVIEW

**Comment by pliu2781** [comment]
Something in this commit is causing the mongo-fs tests to fail.  I will try to find where it is

File: regolith/schemas.py [hunk 1]
```diff
@@ -10,17 +11,20 @@
 
 ACTIVITIES_TYPES = ["teaching", "research"]
 AGENCIES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused", "cancelled"]
```
Comment (sbillinge) [inline regolith/schemas.py:14]: please remove.   These comments may have gotten lost on another PR perhaps, but this is not the right pattern.  You should be importing the `PROJECTA` stati into the progress report.  Don't duplicate them with new non-specific names.

**Review by sbillinge — COMMENTED** [review]
thanks so much for adding these tests, it is great!  because it is alister, we can, in principle, add multiple tests to test all lines of code.  Codecov report can help to show which lines are not covered.

I also left a comment or two inline.

**Comment by pliu2781** [comment]
I found the travis issue, it was a datetime object that I put in schemas that caused something to fail. I fixed it on the original PR, so I am closing this one

**Pull request closed without merging**